packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -915,23 +915,23 @@ async function determinePackageBasedOrIntegratedOrStandalone(): Promise<
     {
       type: 'autocomplete',
       name: 'workspaceType',
-      message: `Package-based monorepo, integrated monorepo, or standalone project?`,
+      message: 'npm/yarn monorepo, Nx monorepo, or standalone project?',
       initial: 0,
       choices: [
         {
           name: 'package-based',
           message:
-            'Package-based Monorepo:     Nx makes it fast, but lets you run things your way.',
+            'Package-based Monorepo:     Use a npm/yarn workspace, but have Nx manage the build',
```

Comment:
Can we say something along the lines of.. Uses a npm/yarn workspace.. but better with Nx?

```
Use npm/yarn workspaces and Nx will make it fast.
```

Reply:
I don't think we should have to do `npm/yarn`, we know the package manager we are scaffolding the workspace for